pkg/cmd/discover.go

```diff
@@ -85,6 +98,17 @@ func (o *DiscoverCmdOptions) Run(_ *cobra.Command) error {
 	}
 
 	discoveredFeatures := features.Discover(config.New(), userPolicies)
+
+	if o.mcpConfig != "" {
+		var mcpConfigProvider api.MCPConfig
+		if o.mcpConfig == "cursor" {
+			mcpConfigProvider = &cursor.CursorMcpConfig{}
+		} else {
+			return fmt.Errorf("invalid editor name '%s', must be one of (%s)", o.mcpConfig, strings.Join(editors, ", "))
+		}
+		return mcpconfig.Save(mcpConfigProvider, discoveredFeatures.Tools)
```

Comment:
In the future we might want to maintain a master mapping somewhere else.
Probably populated with an inverse registration (similar to what we do with features -inference,tools-).